Installationsskripte + Proxysettings

ba-lindner opened the pull request:
- meine Skripte
- Proxy-Einstellungen für build

ba-lindner committed:
Installationsskripte + Proxysettings

- meine Skripte
- Proxy-Einstellungen für build

niklas-simon commented:
# Review
## Dockerfile
- Proxy-Einstellungen in Ordnung
- ENV-Variablen bitte vollständig entfernen, statt auszukommentieren
## compose.yml
- festes Setzen der ENV-Variablen nicht ok, dafür gibt es das .env-File
- healthcheck in Ordnung
- network wird automatisch erstellt, warum jetzt manuell?
- proxy muss ich noch testen
## *.sh
- muss ich noch testen

niklas-simon commented:
closes #1 through healthcheck

niklas-simon reviewed (changes requested):
# Review
## Dockerfile
- Proxy-Einstellungen in Ordnung
- ENV-Variablen bitte vollständig entfernen, statt auszukommentieren
## compose.yml
- festes Setzen der ENV-Variablen nicht ok, dafür gibt es das .env-File
- healthcheck in Ordnung
- network wird automatisch erstellt, warum jetzt manuell?
- proxy muss ich noch testen
## *.sh
- muss ich noch testen

ba-lindner committed:
fixed some problems

- Proxy-Einstellungen zusammengefasst
- Environment-Variablen aus Dockerfile entfernt
- Netzwerk in compose entfernt
- env_file in compose  wieder hinzugefügt
- readme angepasst

niklas-simon committed:
crlf -> lf

ba-lindner committed:
Skripte aktualisiert

- Skripte auf LF statt CRLF umgestellt
- Readme aktualisiert
- Nach der Installation wird gefragt, ob der Rechner neugestartet werden soll

ba-lindner committed:
Merge branch 'bernhard' of https://github.com/niklas-simon/cat-database into bernhard

niklas-simon committed:
crlf -> lf

niklas-simon committed:
fix docker installation command

niklas-simon committed:
non-root setup.sh

niklas-simon commented:
nach ein paar Anpassungen meinerseits funktioniert alles

niklas-simon reviewed (approved):
nach ein paar Anpassungen meinerseits funktioniert alles
